react/components/atoms/container-swipeable/container-swipeable.js

```diff
@@ -1,173 +1,252 @@
 import React, { PureComponent } from "react";
-import { Animated, StyleSheet, Modal, View, TouchableOpacity } from "react-native";
-import { initialWindowSafeAreaInsets } from "react-native-safe-area-context";
+import {
+    ViewPropTypes,
+    StyleSheet,
+    View,
+    Dimensions,
+    TouchableOpacity,
+    Animated,
+    Easing,
+    StatusBar,
+    Platform
+} from "react-native";
 import PropTypes from "prop-types";
+import Modal from "react-native-modal";

```

Comment:
@NFSS10 do we really need this? Can't we use `Modal` from `react-native` like we did before?

Reply:
This was used to get the gestures out of the box, but then the swipe gestures were made to be the same code for fullscreen and not in fullscreen mode.

In summary yes, I think we could use `Modal `from` react-native` like we did before. I will make the changes